docs/api/rbac/rbac-example.mdx

```diff
@@ -0,0 +1,81 @@
+---
+sidebar_position: 2
+title: Create RBAC policy
+---
+
+### Example #1
+
+#### We have a shared todo app that allows users to create, read, update, toggle and delete tasks.
+
+First, lets define our roles resources and permissions.
+
+**Roles** - Admin and Operator.<br/>
+**Resource** - Task with create, read, update, toggle and delete actions.<br/>
+**Permissions** - Admin can do everything and Operator can only read and toggle tasks.<br/>
+
+Now let's create a Roles and Resources and Role Permissions through the API.<br/>
+But before we need to get our `API_SECRET_KEY` from the dashboard and get current `project_id` and `env_id`<br/>
+- [Follow this link to get your API_SECRET_KEY](/api/api-with-cli/#get-your-api-key).<br/>
+- [To get your project_id and env_id follow this link](/api/examples/get-project-and-env).
+
+After you got your `API_SECRET_KEY`, `project_id` and `env_id` you can create your roles, resources and role permissions.<br/>
+Make sure to replace `API_SECRET_KEY`, `project_id` and `env_id` with your own values project_id and project_key are interchangeable the same apply to env_id and env_key.
+
+
+```bash
+# create an admin role
+curl 'https://api.permit.io/v2/schema/project_id/env_id/roles' \
```

Comment:
Can we have options for other languages? I've created a component where you can toggle different languages. It does need to be improved to support rendering markdown better - but maybe you want to have a stab at it?

Reply:
I didn't change this file, so if we want to work on that I can ask Or and add a task on that.
But I don't want to block the API docs on that..